package.json

```diff
@@ -13,12 +13,13 @@
     "build": "node_modules/.bin/parallel-webpack",
     "doc": "node_modules/.bin/jsdoc -c jsdoc-conf.json",
     "test": "node_modules/.bin/karma start --single-run --browsers=ChromeHeadless",
-    "test:all": "for pkg in core jquery jquery-file-upload; do node_modules/.bin/karma start --single-run --cloudinary.pkg=${pkg}; done",
-    "test:all:prod": "for pkg in core jquery jquery-file-upload; do node_modules/.bin/karma start --single-run --cloudinary.minified --cloudinary.pkg=${pkg}; done",
+    "test:pkgs": "for pkg in core jquery jquery-file-upload; do node_modules/.bin/karma start --single-run --cloudinary.pkg=${pkg}; done",
+    "test:pkgs:minified": "for pkg in core jquery jquery-file-upload; do node_modules/.bin/karma start --single-run --cloudinary.minified --cloudinary.pkg=${pkg}; done",
     "test:types": "node_modules/.bin/tsc",
+    "test:pkgs:verify": "jasmine test/spec/pkg-spec.js",
     "pretest": "npm run build",
-    "pretest:all": "npm run build",
-    "pretest:all:prod": "npm run build",
+    "pretest:pkgs": "npm run build",
```

Comment:
@maoznir , again, why do we need those?

Reply:
These make sure a build is done before the test.
npm runs them automatically before scripts with the same name without the "pre" prefix.
It's not a must but a convenience. anyway they were not touched in this pr except for matching the name to each related script.